packages/backend/src/graphql/resolvers/tenant_settings.ts

```diff
@@ -24,27 +17,11 @@ export const getTenantSettings: TenantResolvers<TenantedApolloContext>['settings
       'tenantSettingService'
     )
 
-    const { sortOrder, ...pagination } = args
-    const order = sortOrder === 'ASC' ? SortOrder.Asc : SortOrder.Desc
+    const tenantSettings = (await tenantSettingsService.get({
+      tenantId: parent.id
+    })) as TenantSetting[]
```

Comment:
```suggestion
    const tenantSettings = await tenantSettingsService.get({
      tenantId: parent.id
    })
```

Reply:
ha yes, good catch. I need to check if Bogdan changed the return type and rebase to our MT base branch. After that it should be ok